.build/tar/generate_tar.sh

```diff
@@ -11,22 +13,18 @@ then
     exit 1
 fi
 
-# check dist folder
-if [ ! -d "dist" ]
-then
-    echo "Not found: dist folder, creating the folder dist"
-    mkdir dist
-fi
```

Comment:
Isn't that still needed? (could be `mkdir -p dist`)

What if `package-tarball` is run directly?